### PR TITLE
[RD-176] Add Author's year of death

### PR DESCRIPTION
**Pull request by ddhogan**
- Validate that it is a larger number than the year of birth, if present

### CODE REVIEW

File: client/src/views/AuthorIndividualForm.vue [hunk 1]
```diff
@@ -138,14 +138,31 @@
...
+              name="author-year-of-death"
+              id="author-year-of-death"
+              type="number"
+              v-model="form.tmpAuthorYearOfDeath"
+              :disabled="sending"
+              @input="updateField()"
+            />
+            <span
+              class="md-error"
+              v-if="!customValidationFields.tmpAuthorYearOfDeath.invalid"
```
Comment (lipsotiko) [inline client/src/views/AuthorIndividualForm.vue]: You did not use the customValidationFields, you used the validations "hook", so this should be something like v-if"!$v.form.customMin"

**Comment by lipsotiko** [comment]
The layout of the form is off, run it locally and see what it looks like. Also, it also won't get submitted when there are no errors and the years are both populated (Birth and Death), that should be covered with a test

File: app/api.py [hunk 2]
```diff
@@ -43,6 +43,7 @@ class CopyrightApplicationModelApi(ModelRestApi):
         'domicile',
         'claimant_organization',
         'claimant_organization_name',
+        'author_year_of_death',
```
Comment (lipsotiko) [inline app/api.py]: Please group with other author fields.

[the remark this answers is not in the document]
Comment (ddhogan) [reply app/api.py]: Thanks for catching that!

Comment (ddhogan) [reply app/api.py]: fixed in commit 1241a7d

[the remark this answers is not in the document]
Comment (ddhogan) [reply app/api.py]: fixed in commit 1241a7d

[the remark this answers is not in the document]
Comment (ddhogan) [reply app/api.py]: fixed in commit 1241a7d

File: client/src/views/AuthorIndividualForm.vue [hunk 3]
```diff
@@ -139,6 +139,23 @@
...
+            name="author-year-of-death"
+            id="author-year-of-death"
+            type="number"
+            v-model="form.tmpAuthorYearOfDeath"
+            :disabled="sending"
+            @input="updateField()"
+          />
+          <span
+            class="md-error"
+            v-if="!$v.form.tmpAuthorYearOfDeath.yearOfDeathValiation"
```
Comment (lipsotiko) [inline client/src/views/AuthorIndividualForm.vue]: minYearOfDeath seems like a better method name based on what I've seen with vuelidate

File: client/src/views/AuthorIndividualForm.vue [hunk 4]
```diff
@@ -295,6 +326,7 @@ export default {
       }
 
       if (this.$v.$invalid || authorNameInvalid || authorCitizenshipDomicileInvalid) {
+        this.invalid = true
```
Comment (lipsotiko) [inline client/src/views/AuthorIndividualForm.vue]: There is a duplicate code block above that does the same thing

File: client/src/views/AuthorIndividualForm.vue (hunk 3 above)
Comment (ddhogan) [reply client/src/views/AuthorIndividualForm.vue]: agreed, fixing

File: client/src/views/AuthorIndividualForm.vue (hunk 4 above)
Comment (ddhogan) [reply client/src/views/AuthorIndividualForm.vue]: I'll remove the one above it

**Pull request merged**